plugins/woocommerce-admin/client/customize-store/assembler-hub/sidebar/global-styles/color-palette-variations/constants.ts

```diff
@@ -37,20 +37,20 @@ export const COLOR_PALETTES = [
 			},
 		},
 		styles: {
-			color: {
-				background: 'var(--wp--preset--color--background)',
-				text: 'var(--wp--preset--color--foreground)',
+			blocks: {
```

Comment:
Is it necessary to remove the global `color` rules here?

It seems to work fine with them in place and I wonder if this would have an unintended impact, considering it's not targeted _specifically_ at the button styles we're aiming to adjust.

The key component is setting the `core/button` block to an empty object, as I understand it.

Reply:
It makes sense! I restored it!